data-plane/python-bindings/examples/Dockerfile

```diff
@@ -1,13 +1,16 @@
 # Copyright AGNTCY Contributors (https://github.com/agntcy)
 # SPDX-License-Identifier: Apache-2.0
 
-FROM ghcr.io/astral-sh/uv:0.7.0-python3.13-bookworm AS builder
+# Build container with cross-compilation support
+FROM --platform=${BUILDPLATFORM} ghcr.io/astral-sh/uv:0.7.0-python3.13-bookworm AS builder
 
-ENV PATH="/root/.cargo/bin:${PATH}"
+SHELL ["/bin/bash", "-c"]
+
+ARG TARGETARCH
 
+ENV PATH="/root/.cargo/bin:${PATH}"
 ENV PYTHONUNBUFFERED=1
 ENV PYTHONDONTWRITEBYTECODE=1
-
 ENV UV_LINK_MODE=copy \
     UV_COMPILE_BYTECODE=1 \
     UV_PYTHON_DOWNLOADS=never \
```

Comment:
We can remove the `UV_PYTHON_DOWNLOADS=never`, as we'll be installing a python version suitable for cross compilation.

Reply:
done